<h1>Contributions</h1>

<hr />

<section>
  <h2>Cyanobacteria in the Registry</h2>
  <p>
    We created new parts for future iGEM teams choosing to work with cyanobacteria. This model organism is ideal for
    studying photosynthesis due to its well characterized metabolic pathways and natural competency that readily allows
    transformation. Cyanobacteria are relatively low-cost and easy to maintain, as well as extremely safe experimental
    organisms as highlighted on our <a href="/Safety">Safety</a> page. Overall, these features make cyanobacteria an excellent choice as a
    model or chassis, especially in synthetic biology.
  </p>
  <p>
    The iGEM Registry currently has few parts specific for cyanobacteria, contrasting with the well characterized parts for
    <em>Escherichia coli</em> and <em>Saccharomyces cerevisiae</em>. This lack of available constructs for cyanobacteria
    limits future photosynthetic-focused iGEM work, as these heterotrophic organism parts are usually not as effective
    or applicable when applied to cyanobacteria. This shortage requires major expansion, reorganization, and
    characterization of parts and tools to expand future iGEM opportunities and photosynthetic research.
  </p>
  <p>
    As part of our project, we created three overexpression plasmids and one deletion plasmid, detailed below. Within
    these plasmids, consisting of several composite parts, we added numerous basic parts involving regulatory and
    expression functions within cyanobacteria. All our registered parts are detailed below in contextual descriptions,
    with technical details being on the linked registry pages.
  </p>
</section>

<section>
  <h2>Parts Registered</h2>

  <section>
    <h3>Overexpression Plasmids</h3>
    <p>
      Our overexpression plasmids were created via Gibson Assembly, of which we registered all relevant parts. These
      plasmids were all used within <em>Synechococcus elongatus</em> PCC 7942 with intended alterations to the
      Calvin-Bensen-Bassham (CBB) cycle.
    </p>
    <p>
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047000">BBa_K4047000</a> and
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047001">BBa_K4047001</a> are primers for amplifying
      the transaldolase (<em>tal</em>) gene (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047034"
        >BBa_K4047034</a
      >) from wild type <em>S. elongatus </em>PCC 7942. The primer contains EcoRI and BamHI restriction enzyme sites for
      confirmation testing. Transaldolase participates in the CBB cycle by catalyzing the forward and reversion
      conversion of sedoheptulose-1,7-bisphosphate and glyceraldehyde-3-phosphate into erythrose-4-phosphate and
      fructose-1,6-bisphosphate. In a similar fashion,
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047002">BBa_K4047002</a> and
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047003">BBa_K4047003</a> are primers for
      amplification of the fructose-1,6-bisphosphatase (<em>fbp</em>) gene (<a
        href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047035"
        >BBa_K4047035</a
      >). They also contained the EcoRI and BamHI restriction enzyme sites. Fructose-1,6-bisphosphatase catalyzes the
      hydrolysis of a phosphate group from fructose-1,6-bisphosphate. This process, part of the CBB cycle, is also vital
      to many other central metabolic pathways.
    </p>
    <p>
      Additional primers were designed so that both tal and fbp, once amplified, could be inserted in a third
      overexpression plasmid (<a href="http://bba_k4047004">BBa_K4047004</a> and
      <a href="http://bba_k4047005">BBa_K4047005</a>).
    </p>
    <p>
      All our overexpression plasmids used pAM2991 as the expression vector (<a
        href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047024"
        >BBa_K4047024</a
      >). This vector was created by Susan Golden in 2005 specifically for <em>S. elongatus</em> PCC 7942, and is
      publically available on <a href="https://www.addgene.org/40248/">addgene.org</a>. Its multiple components include
      an origin of replication (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047029">BBa_K4047029</a
      >), a trc promoter (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047032">BBa_K4047032</a>), an
      rrnB T1 terminator (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047025">BBa_K4047025</a>), an
      rrnB T2 terminator (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047026">BBa_K4047026</a>), a
      base of mobility region (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047028">BBa_K4047028</a>),
      a lac repressor (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047031">BBa_K4047031</a>), a lacIq
      promoter (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047030">BBa_K4047030</a>), a lac operator
      (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047033">BBa_K4047033</a>), and
      spectinomycin/streptomycin resistance (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047027"
        >BBa_K4047027</a
      >). Sequences homologous to the neutral site I region allow for these components along with our target genes to be
      inserted into the genome of <em>S. elongatus</em> PCC 7942.
    </p>
    <p>
      Our first plasmid, called pGEM-tal (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047036"
        >BBa_K4047036</a
      >), aimed to induce transaldolase overexpression by inserting an additional copy of the encoding gene,
      <em>tal</em> (Figure 1).
    </p>
    <span class="image half-full-width left"
      ><img src="assets/images/contributions/figure-1.png" alt="Annotated SnapGene viewer of pGEM-tal." /><span class="credit"
        >Figure 1: Annotated SnapGene viewer of pGEM-tal.</span
      ></span
    >
    <p>
      Our second plasmid, named pGEM-fbp (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047037"
        >BBa_K4047037</a
      >), induced fructose-1,6-bisphosphate overexpression by inserting an additional copy of the encoding
      <em>fbp</em> gene (Figure 2).
    </p>
    <span class="image half-full-width left"
      ><img src="assets/images/contributions/figure-2.png" alt="Annotated SnapGene viewer of pGEM-fbp." /><span class="credit"
        >Figure 2: Annotated SnapGene viewer of pGEM-fbp.</span
      ></span
    >
    <p>
      Finally, our third overexpression plasmid, pGEM-tal+fbp (<a
        href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047038"
        >BBa_K4047038</a
      >), combined the encoding elements of the above overexpression plasmids, aiming to insert both <em>tal</em> and
      <em>fbp</em> genes to overexpress both genes (Figure 3).
    </p>
    <span class="image half-full-width left"
      ><img src="assets/images/contributions/figure-3.png" alt="Annotated SnapGene viewer of pGEM-tal+fbp." /><span class="credit"
        >Figure 3: Annotated SnapGene viewer of pGEM-tal+fbp.</span
      ></span
    >
  </section>

  <section>
    <h3>Deletion Plasmid</h3>
    <p>
      Our deletion plasmid, named pGEM-SBPase (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047039"
        >BBa_K4047039</a
      >), was designed to delete sedoheptulose-1,7-bisphosphate (<em>glpX</em>) from the<em> S. elongatus</em> PCC 7942
      genome through homologous recombination with upstream (<a
        href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047022"
        >BBa_K4047022</a
      >) and downstream (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047023">BBa_K4047023</a>)
      sequences (Figure 4). Two sets of primers (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047009"
        >BBa_K4047009</a
      >
      and <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047010">BBa_K40470010</a>,
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047013">BBa_K4047013</a> and
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047014">BBa_K4047014</a>) were used to amplify 800
      base pair upstream and downstream fragments of <em>glpX</em> from the <em>S. elongatus </em>genome. We designed
      additional primers (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047007">BBa_K4047007</a> and
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047008">Bba_K4047008</a>) to amplify the backbone
      of pXWK3-glgC. This backbone is well-established as a vector within the Wang Microbiology Lab at Miami University,
      used in a <a href="https://academic.oup.com/plphys/article/182/1/507/6116245">previous publication</a>. The
      backbone (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047018">BBa_K4047018</a>) contains an
      origin of replication and a gentamicin resistance gene (<a
        href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047016"
        >BBa_K4047016</a
      >), promoter (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047015">BBa_K4047015</a>), and
      terminator (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047017">BBa_K4047017</a>). In addition,
      we used primers (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047011">BBa_K4047011</a> and
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047012">BBa_K4047012</a>) to amplify a kanamycin
      resistance cassette from pXWK3-glgC (<a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047019"
        >BBa_K4047019</a
      >, <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047020">BBa_K4047020</a>,
      <a href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K4047021">BBa_K4047021</a>).
    </p>
    <span class="image half-full-width left"
      ><img src="assets/images/contributions/figure-4.png" alt="" /><span class="credit"
        >Figure 4: Annotated SnapGene viewer of pGEM-SBPase.</span
      ></span
    >
  </section>

  <section>
    <h3>Part compatibility</h3>
    <p>
      Our parts have a variety of compatibility detailed in Table 1. The vast majority of our parts are compatible with
      all RFC standards. All our plasmids, due to possessing various restriction sites, are not RFC compatible. Several
      of our components, despite having limited RFC compatibility, may be extremely useful for future cyanobacteria
      teams due to their vast array of functions, suitability for the widely adaptable and accessible pAM2991 vector,
      and handling of the central metabolic genes <em>tal </em>and <em>fbp</em>.
    </p>
    <p>Table 1: Complete list of registered parts and their compatibilities within the iGEM registry.</p>
    <div class="table-wrapper">
      <table>
        <thead>
          <tr>
            <th>
              <strong>Parts</strong>
            </th>
            <th>
              <strong>Compatibility</strong>
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>BBa_K4047000, BBa_K4047001, BBa_K4047002, BBa_K4047003</td>
            <td>
              <strong>RFC[1000]</strong>
            </td>
          </tr>
          <tr>
            <td>
              BBa_K4047004, BBa_K4047005, BBa_K4047007, BBa_K4047008, BBa_K4047009, BBa_K4047010, BBa_K4047012,
              BBa_K4047013, BBa_K4047014, BBa_K4047016, BBa_K4047017, BBa_K4047019, BBa_K4047020, BBa_K4047025,
              BBa_K4047026, BBa_K4047028, BBa_K4047029, BBa_K4047030, BBa_K4047031, BBa_K4047032, BBa_K4047033
            </td>
            <td>
              <strong>RFC{10][12][21][23][25][1000]</strong>
            </td>
          </tr>
          <tr>
            <td>BBa_K4047011, BBa_K4047021</td>
            <td>
              <strong>RFC[21][1000]</strong>
            </td>
          </tr>
          <tr>
            <td>BBa_K4047015</td>
            <td>
              <strong>RFC{10][12][21][23][25]</strong>
            </td>
          </tr>
          <tr>
            <td>
              BBa_K4047018, BBa_K4047022, BBa_K4047024, BBa_K4047035, BBa_K4047036, BBa_K4047037, BBa_K4047038,
              BBa_K4047039
            </td>
            <td></td>
          </tr>
          <tr>
            <td>BBa_K4047023, BBa_K4047027</td>
            <td>
              <strong>RFC{10][12][21][23][1000]</strong>
            </td>
          </tr>
          <tr>
            <td>BBa_K4047034</td>
            <td>
              <strong>RFC[10][21][23]</strong>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </section>
</section>

<hr />

<section>
  <h2>Wiki Development</h2>
  <p>
    <span class="image third right"
      ><img
        src="assets/images/about/html.png"
        alt="A website IDE open with the Miami University's home page code on it."
      /><span class="credit">A website editor open with the Miami University's home page code on it.</span></span
    >
    The wiki server has been notorious in iGEM for its difficulty to navigate, and many teams struggle to develop a
    strong wiki by not being familiar with professional website development. Therefore, we sought to create a platform
    using code and philosophies that could guide other teams for easier and more directed success. Better wikis also
    make the important content more accessible to readers, therefore improving upon the goal of scientific communication
    for iGEM.
  </p>
  <p>
    First, some code from previous years’ teams were edited to provide a streamlined uploading process that essentially
    takes word documents, transfers them into markdown files, and uploads them to the wiki server. Many bugs have been
    resolved and unique difficulties often faced by iGEM teams such as uploading complex mathematical notation are now
    documented in easily accessible code. Thorough documentation of changes and additions can be found in our publicly
    accessible Github repository for iGEM-2021-Website under MiamiOH-iGEM.
  </p>
  <p>
    In addition to the natural content of the code, our documentation and intentionality for the Wiki provide a good
    example by which other teams can follow. First, the code related to the wiki is documented in a way that clearly
    expresses how to use the code, with helpful diagrams and a sitemap describing the layout of the site. Exact
    representation of these detailed descriptions and instructions can be found in our publicly accessible Github
    repository for iGEM-2021-Website under MiamiOH-iGEM. Similarly, our wiki design is intentional in having information
    be as accessible as possible. For example, a dictionary was incorporated to allow viewers to simply hover over
    certain terms and be shown a small definition, serving as easy reminders of some of the more technical terms
    propagating the wiki. Similarly, all pages were checked to be formatted correctly on all mobile and desktop devices.
    Rather than describe all philosophies here, we encourage you to look at additional intentions and more thorough
    descriptions of the previously described steps on the
    <a href="/About_The_Wiki">"About the Wiki" Page</a>.
  </p>
</section>

<hr />
<explore pages="Communication,Collaborations" />
