<h1>Notebook</h1>
<p>
  Throughout the course of the wet lab experiments, we kept thorough notes on our protocols and methods. 
  Our ability to troubleshoot and improve our techniques as we proceeded significantly contributed to our eventual success. 
  This page is a recount of the path we took to get there. From primer design in the summer to
  growth experiments in the fall, this iGEM experience showcased the full process of genetic engineering.
</p>

<hr />

<h2>Full Lab Protocol</h2>
<p>
  For a full description of our lab protocols, please see the downloadable notebook below or explore our in-site notebook!
</p>
<a href="assets/pdfs/notebook.pdf" class="customButton icon solid fa-download">Download Notebook</a>

<hr />

<h2>Timeline</h2>
<ul class="accordion">
  <li>
    <a>Fragment Amplification (29 July - 2 August)</a>
    <section>
      <p>
        We used polymerase chain reaction (PCR) to amplify fragments that would be used for Gibson Assembly of our
        plasmids.
      </p>
      <p>
        For the overexpression plasmids -pGEM-tal, pGEM-fbp, and pGEM-tal+fbp- we needed to amplify the
        <i>transaldolase</i> and <i>fructose-1,6-bisphosphatase</i> (class I) genes from the
        <i>Synechococcus elongatus </i>PCC 7942 genome.
      </p>
      <p>
        For the gene deletion plasmid- pGEM-SBPase- we amplified the backbone of pXWK3_glgC, a kanamycin resistance
        (kanR) cassette, and ~800 bp of both an upstream and downstream fragment of sedoheptulose-1,7-bisphosphatase
        (SBPase) from the PCC 7942 genome.
      </p>
    </section>
  </li>
  <li>
    <a>Fragment Amplification (3 August)</a>
    <section>
      <p>We took our fragments straight from the gel.</p>
    </section>
  </li>
  <li>
    <a>Fragment Amplification (4 August)</a>
    <section>
      <p>The backbone for the overexpression plasmids was restriction digested from pAM2991, not PCR amplified.</p>
  </section>
  </li>
  <li>
    <a>Gibson Assembly (5 August)</a>
    <section>
      <p>
        The fragments above have overlapping sequences with what they are next to on the plasmid. The primers we
        designed are what allowed for this. The overlap was used during the Gibson Assembly of our plasmids.
      </p>
      <p>The overlap was used during the Gibson Assembly of our plasmids.</p>
    </section>
  </li>
  <li>
    <a>Transformation of <i>E. coli </i> (5 August)</a>
    <section>
      <p>
        We took our assembled plasmids and used chemical transformation to put them in
        <i>Escherichia coli </i>dh10ùõÉ. <i>E. coli</i> will replicate the plasmids to give us multiple copies.
        <i>S. elongatus </i>doesn‚Äôt do this.
      </p>
      <p>
        Antibiotic resistance in the plasmid means we can grow cells on growth media containing antibiotics to pick
        out successful transformants.
      </p>
    </section>
  </li>
  <li>
    <a>Purification of plasmids from <i>E. coli</i> (9 August)</a>
    <section>
      <i></i>
      <p>We needed to purify our plasmids from <i>E. coli </i>before transforming <i>S. elongatus</i>.</p>
      <p>We broke up the plasmid (restriction digest) to make sure our genes were in there.</p>
    </section>
  </li>
  <li>
    <a>Transformation of <i>S. elongatus </i>PCC 7942 (10 August - 7 September)</a>
    <section>
      <p>
        Using multiple copies of our plasmids from <i>E. coli</i>, we could expect more efficient transformation of
        <i>S. elongatus</i>.
      </p>
      <p>Transformation means <i>S. elongatus </i>would take in our plasmids and integrate them into its genome.</p>
      <p>We used antibiotics to pick out successful transformants.</p>
    </section>
  </li>
  <li>
    <a>Transformation of <i>S. elongatus </i>PCC 7942 (23 August - 20 September)</a>
    <section>
      <p>
        Individual colonies from the original transformation plate were picked out later and transferred onto a fresh
        plate. They get more space and resources to grow better this way.
      </p>
    </section>
  </li>
  <li>
    <a>Selection of double crossover for deletion of SBPase and PCR confirmation (23 August)</a>
    <section>
      <p>
        Our experiment with deleting the SBPase gene simply proved again that it was essential and that you can‚Äôt get
        SBPase deleted in all copies of the genome.
      </p>
      <p>
        To get proper deletion though the whole plasmid can‚Äôt be what gets into the genome. We have a copy of SBPase
        in our plasmid that is interrupted by kanamycin so that it is nonfunctional. This is what should replace the
        native SBPase.
      </p>
      <p>
        We had to select colonies that did not have gentamicin resistance, which is in the plasmid backbone, but had
        kanamycin resistance, which is what SBPase is interrupted by.
      </p>
    </section>
  </li>
  <li>
    <a>Segregation PCR for overexpression colonies (tal, fbp, tal-fbp) (30 August - 11 October)</a>
    <section>
      <p>
        We used this step to see that the gene- tal, fbp, or both- were integrated into the genome. The specific
        integration site was called neutral site I (NSI).
      </p>
      <p>
        We lysed/broke open cells from the transferred transformants and performed polymerase chain reaction to
        amplify what was in NSI.
      </p>
      <p>
        An important note here is that <i>S. elongatus </i>has multiple copies of its genome so our gene could be in
        the NSI of some copies and not others.
      </p>
      <p>
        We grew up the transferred transformants in liquid cultures eventually. This was small-scale in 6-well plates.
        Our segregation PCR hadn‚Äôt been getting good results up to this point so putting them in liquid cultures and
        letting them grow more this way would help us.
      </p>
      <p>
        We did the segregation PCR for the cultures from the 6-well plates. The cells/colonies which showed the least
        copies of ‚Äúwildtype‚Äù genome and/or the most copies of transformed NSI were ‚Äúscaled up‚Äù to a higher volume and
        kept in a 250-mL flask.
      </p>
    </section>
  </li>
  <li>
    <a>Growth Experiments (7 October - 13 October)</a>
    <section>
      <p>
        We ended up using the liquid cultures from the 6-well plates for these experiments. We chose 3 cultures each
        from tal, tal-fbp, and fbp (the ones that we confirmed had the transformed NSI).
      </p>
      <p>
        We scaled up these cultures to a 50 mL solution and used a chemical inducer (IPTG) to get even more
        overexpression of tal, fbp, or both.
      </p>
      <p>
        We kept these colonies growing in the liquid culture under 10 hr light and 14 hr dark cycles and collected
        optical density measurements.
      </p>
      <p>
        The optical density measures are simply just using the color of the solution (the more green= the more cells)
        to see the growth over time.
      </p>
    </section>
  </li>
</ul>

<hr />
<explore pages="Proof_Of_Concept,Implementation" />
