<h1>Notebook</h1>
<p>
  Praesent ac adipiscing ullamcorper siper ut amet ac risus. Lori sapien ut odio odio nunc. Ac adipiscing nibh porttitor
  erat risus justo adipiscing adipiscing amet placerat accumsan. Vis. Faucibus odio magna tipus adipiscing a non. In mi
  primis arcu ut non accumsan vivamus ac blandit adipiscing adipiscing arcu metus praesent turpis eu ac lacinia nunc ac
  commodo gravida adipiscing eget accumsan ac nunc adipiscing adipiscing lori ipsum dolor sit amet nullam veroeros
  adipiscing.
</p>

<hr />

<h2>Full Lab Protocol</h2>
<p>
  Praesent ac adipiscing ullamcorper siper ut amet ac risus. Lori sapien ut odio odio nunc. Ac adipiscing nibh porttitor
  erat risus justo adipiscing adipiscing amet placerat accumsan. Vis. Faucibus odio magna tipus adipiscing a non. In mi
  primis arcu ut non accumsan vivamus ac blandit adipiscing adipiscing arcu metus praesent turpis eu ac lacinia nunc ac
  commodo gravida adipiscing eget accumsan ac nunc adipiscing adipiscing lori ipsum dolor sit amet nullam veroeros
  adipiscing.
</p>
<a href="#" class="customButton icon solid fa-download">Download Notebook</a>

<hr />

<h2>Interactable Timeline</h2>
<div class="cd-horizontal-timeline">
  <div class="timeline">
    <div class="events-wrapper">
      <div class="events">
        <ol>
          <li><a href="#0" data-date="29/07/2021" class="selected">29 Jul</a></li>
          <li><a href="#0" data-date="03/08/2021">3 Aug</a></li>
          <li><a href="#0" data-date="04/08/2021">4 Aug</a></li>
          <li><a href="#0" data-date="05/08/2021">5 Aug</a></li>
          <li><a href="#0" data-date="09/08/2021">9 Aug</a></li>
          <li><a href="#0" data-date="10/08/2021">10 Aug</a></li>
          <li><a href="#0" data-date="23/08/2021">23 Aug</a></li>
          <li><a href="#0" data-date="30/08/2021">30 Aug</a></li>
          <li><a href="#0" data-date="7/09/2021">7 Oct</a></li>
        </ol>

        <span class="filling-line" aria-hidden="true"></span>
      </div>
      <!-- .events -->
    </div>
    <!-- .events-wrapper -->

    <ul class="cd-timeline-navigation">
      <li><a href="#0" class="prev inactive">Prev</a></li>
      <li><a href="#0" class="next">Next</a></li>
    </ul>
    <!-- .cd-timeline-navigation -->
  </div>
  <!-- .timeline -->

  <div class="events-content">
    <ol>
      <li class="selected" data-date="29/07/2021">
        <h2>Fragment Amplification</h2>
        <i>29 July 2021 - 2 August 2021</i>
        <p>
          We used polymerase chain reaction (PCR) to amplify fragments that would be used for Gibson Assembly of our
          plasmids.
        </p>
        <p>
          For the overexpression plasmids -pGEM-tal, pGEM-fbp, and pGEM-tal+fbp- we needed to amplify the
          <i>transaldolase</i> and <i>fructose-1,6-bisphosphatase</i> (class I) genes from the
          <i>Synechococcus elongatus </i>PCC 7942 genome.
        </p>
        <p>
          For the gene deletion plasmid- pGEM-SBPase- we amplified the backbone of pXWK3_glgC, a kanamycin resistance
          (kanR) cassette, and ~800 bp of both an upstream and downstream fragment of sedoheptulose-1,7-bisphosphatase
          (SBPase) from the PCC 7942 genome.
        </p>
      </li>

      <li data-date="03/08/2021">
        <h2>Fragment Amplification</h2>
        <i>3 August 2021</i>
        <p>We took our fragments straight from the gel</p>
      </li>

      <li data-date="04/08/2021">
        <h2>Fragment Amplification</h2>
        <i>4 August 2021</i>
        <p>The backbone for the overexpression plasmids was restriction digested from pAM2991, not PCR amplified</p>
      </li>

      <li data-date="05/08/2021">
        <h2>Gibson Assembly</h2>
        <i>5 August 2021</i>
        <p>
          The fragments above have overlapping sequences with what they are next to on the plasmid. The primers we
          designed are what allowed for this. The overlap was used during the Gibson Assembly of our plasmids
        </p>
        <p>The overlap was used during the Gibson Assembly of our plasmids.</p>

        <br />

        <h2>Transformation of <i>E. coli </i></h2>
        <i>5 August 2021</i>
        <p>
          We took our assembled plasmids and used chemical transformation to put them in
          <i>Escherichia coli </i>dh10ùõÉ. <i>E. coli</i> will replicate the plasmids to give us multiple copies.
          <i>S. elongatus </i>doesn‚Äôt do this.
        </p>
        <p>
          Antibiotic resistance in the plasmid means we can grow cells on growth media containing antibiotics to pick
          out successful transformants.
        </p>
      </li>

      <li data-date="09/08/2021">
        <h2>Purification of plasmids from <i>E. coli</i></h2>
        <i>9 August 2021</i>
        <p>We needed to purify our plasmids from <i>E. coli </i>before transforming <i>S. elongatus</i>.</p>
        <p>We broke up the plasmid (restriction digest) to make sure our genes were in there.</p>
      </li>

      <li data-date="10/08/2021">
        <h2>Transformation of <i>S. elongatus </i>PCC 7942</h2>
        <i>10 August 2021 - 7 September 2021</i>
        <p>
          Using multiple copies of our plasmids from <i>E. coli</i>, we could expect more efficient transformation of
          <i>S. elongatus</i>.
        </p>
        <p>Transformation means <i>S. elongatus </i>would take in our plasmids and integrate them into its genome.</p>
        <p>We used antibiotics to pick out successful transformants.</p>
      </li>

      <li data-date="23/08/2021">
        <h2>Transformation of <i>S. elongatus </i>PCC 7942</h2>
        <i>23 August 2021 - 20 September 2021</i>
        <p>
          Individual colonies from the original transformation plate were picked out later and transferred onto a fresh
          plate. They get more space and resources to grow better this way
        </p>

        <br />

        <h2>Selection of double crossover for deletion of SBPase and PCR confirmation</h2>
        <i>23 August 2021</i>
        <p>
          Our experiment with deleting the SBPase gene simply proved again that it was essential and that you can‚Äôt
          get SBPase deleted in all copies of the genome
        </p>
        <p>
          To get proper deletion though the whole plasmid can‚Äôt be what gets into the genome. We have a copy of SBPase
          in our plasmid that is interrupted by kanamycin so that it is nonfunctional. This is what should replace the
          native SBPase.
        </p>
        <p>
          We had to select colonies that did not have gentamicin resistance, which is in the plasmid backbone, but had
          kanamycin resistance, which is what SBPase is interrupted by.
        </p>
      </li>

      <li data-date="30/08/2021">
        <h2>Segregation PCR for overexpression colonies (tal, fbp, tal-fbp)</h2>
        <i>30 August 2021 - 11 October</i>
        <p>
          We used this step to see that the gene- tal, fbp, or both- were integrated into the genome. The specific
          integration site was called neutral site I (NSI).
        </p>
        <p>
          We lysed/broke open cells from the transferred transformants and performed polymerase chain reaction to
          amplify what was in NSI.
        </p>
        <p>
          An important note here is that <i>S. elongatus </i>has multiple copies of its genome so our gene could be
          in the NSI of some copies and not others.
        </p>
        <p>
          We grew up the transferred transformants in liquid cultures eventually. This was small-scale in 6-well
          plates. Our segregation PCR hadn‚Äôt been getting good results up to this point so putting them in liquid
          cultures and letting them grow more this way would help us.
        </p>
        <p>
          We did the segregation PCR for the cultures from the 6-well plates. The cells/colonies which showed the
          least copies of ‚Äúwildtype‚Äù genome and/or the most copies of transformed NSI were ‚Äúscaled up‚Äù to a higher
          volume and kept in a 250-mL flask.
        </p>
      </li>

      <li data-date="7/09/2021">
        <h2>Growth Experiments</h2>
        <i>7 October 2021 - 13 October 2021</i>
        <p>
          We ended up using the liquid cultures from the 6-well plates for these experiments. We chose 3 cultures each
          from tal, tal-fbp, and fbp (the ones that we confirmed had the transformed NSI).
        </p>
        <p>
          We scaled up these cultures to a 50 mL solution and used a chemical inducer (IPTG) to get even more
          overexpression of tal, fbp, or both.
        </p>
        <p>
          We kept these colonies growing in the liquid culture under 10 hr light and 14 hr dark cycles and collected
          optical density measurements.
        </p>
        <p>
          The optical density measures are simply just using the color of the solution (the more green= the more
          cells) to see the growth over time.
        </p>
      </li>

    </ol>
  </div>
</div>

<hr />
<explore pages="Results,Implementation" />
