<section>
  <h1>Proof of Concept</h1>
  <h2>Background</h2>
  <p>
    Crop production is not keeping up with the demands of a growing population and as a consequence, large regions of
    people are left malnourished. In order to prevent a worldwide crisis in the coming decades, we must strengthen our
    current biological tools in an efficient but responsible manner. Through our project, we have demonstrated that
    overexpressing key Calvin-Benson-Bassham (CBB) cycle enzymes improves the viability of cyanobacterial cells (<em
      >Synechococcus elongatus </em
    >PCC 7942)- a photosynthetic organism. We hope that our research serves as a basis for further optimization in
    energy and metabolite flux of photosynthesis, and that this technology is used to improve crop production.
  </p>
</section>

<section>
  <h2>Experimentation and General Methods</h2>
  <h3>Initial Amplification of Plasmid Parts</h3>
  <p>
    After careful primer design, the set of fragments for four different vector constructs were amplified through
    polymerase chain reaction (PCR).
  </p>
  <p>The fragments were:</p>
  <ol>
    <li>the <em>transaldolase</em> (<em>tal</em>) gene for pGEM-tal;</li>
    <li>the <em>fructose-1,6-bisphosphatase</em> (<em>fbp</em>) Class I gene for pGEM-fbp;</li>
    <li>
      both the <em>tal</em><strong><em> </em></strong>and <em>fbp</em> genes for pGEM-tal+fbp;
    </li>
    <li>
      and kanamycin resistance (kmR), upstream and downstream sequences of
      <em>sedoheptulose-1-7-bisphosphatase </em>(<em>glpX</em>), and backbone for pGEM-SBPase
    </li>
  </ol>
</section>

<section>
  <h3>Assembly of Fragments through Gibson Assembly</h3>
  <p>
    The amplified fragments were designed with recognition sites for endonucleases used in Gibson Assembly. We
    successfully assembled and verified through restriction digest four vector constructs- pGEM-Tal, pGEM-FBPase,
    pGEM-tal+fbp, and pGEM-SBPase (Figures 1-4). The first three plasmids were built for overexpression of the
    respective CBB cycle enzymes. The final plasmid listed was designed for knocking out <em>glpX</em> in the
    <em>S. elongatus </em>PCC7942 genome.
  </p>

  <p>
    <span class="image half-full-width left"
      ><img src="assets/images/proof-of-concept/figure-1.png" alt="Viewer plasmid map of pGEM-tal." /><span
        class="credit"
      >
        Figure 1: SnapGene¬Æ Viewer plasmid map of pGEM-tal
      </span></span
    >
  </p>

  <p>
    <span class="image half-full-width left"
      ><img src="assets/images/proof-of-concept/figure-2.png" alt="Viewer plasmid map of pGEM-fbp" /><span
        class="credit"
      >
        Figure 2: SnapGene¬Æ Viewer plasmid map of pGEM-fbp.
      </span></span
    >
  </p>

  <p>
    <span class="image half-full-width left"
      ><img src="assets/images/proof-of-concept/figure-3.png" alt="Viewer plasmid map of pGEM-tal+fbp" /><span
        class="credit"
      >
        Figure 3: SnapGene¬Æ Viewer plasmid map of pGEM-tal+fbp.
      </span></span
    >
  </p>

  <p>
    <span class="image half-full-width left"
      ><img src="assets/images/proof-of-concept/figure-4.png" alt="Viewer plasmid map of pGEM-SBPase" /><span
        class="credit"
      >
        Figure 4: SnapGene¬Æ Viewer plasmid map of pGEM-SBPase.
      </span></span
    >
  </p>
</section>

<section>
  <h3>Transformation of <em>E. coli </em>DH10ùõÉ and Amplification of Assembled Plasmids</h3>
  <p>
    We used chemical transformation of our chassis <em>E. coli</em> DH10ùõÉ to amplify our plasmid constructs. These
    plasmids were purified through miniprep and transformed into our target organism- cyanobacteria
    <em>S. elongatus </em>PCC 7942. <em> </em>
  </p>
</section>

<section>
  <h3>Verification of Desired Constructs through DNA Sequencing</h3>
  <p>
    We followed a cycle sequencing and product purification protocol outlined by the Center for Bioinformatics and
    Functional Genomics CBFG <reference identifier="58" /> of Miami University. The nucleotide pellets were turned in
    for automated sequencing. The results allowed us to verify that the genes and promoters used for overexpression had
    the expected DNA sequences.
  </p>
</section>

<section>
  <h3>Transformation of <em>Synechococcus elongatus</em> PCC 7942</h3>
  <p>
    <em>S. elongatus</em> is naturally competent and doesn‚Äôt require chemical transformation. Once we purified our
    plasmids from <em>E. coli</em>, we followed a simple protocol to transform our plasmids in <em>S. elongatus</em>. We
    used kanamycin to select for target deletion mutants and streptomycin/spectinomycin to select for over expression
    colonies.
  </p>
</section>

<section>
  <h3>Confirmation of Genome Insertion Through Segregation PCR</h3>
  <p>
    We wanted to confirm that the overexpression plasmids integrated into neutral site I or, for deletion mutants, that
    the inactivated SBPase gene inserted in place of the native SBPase. We therefore performed PCR amplification of the
    target insertion sites and looked for our expected band lengths. Just as important was determining the amplicon size
    resulting from the wildtype genome. <em>S. elongatus</em> PCC 7942 has multiple copies of its genome and we can
    therefore see the extent of segregation by looking for wild type (WT) amplicons.
  </p>
</section>

<section>
  <h3>Growth Experiments</h3>
  <p>
    We performed growth experiments on our overexpression colonies and the wildtype strain. We grew the cells in liquid
    cultures under 10 hr light and 14 hour dark cycles for five days. Throughout this period we collected optical
    density data to monitor growth.
  </p>
</section>

<section>
  <h2>Results:</h2>
  <p>
    Cells with individual overexpression of transaldolase (<em>tal</em>) and FBPase (<em>fbp</em>) showed identical
    growth, both showing growth that was less robust compared to the wild type (WT) (Figure 5). Identical phenotypes
    suggest that both FBPase and transaldolase overexpression individually have the same implications on the cells. Both
    transaldolase and FBPase, in their native roles, produce fructose-6-phosphate (F6P). In the case of FBPase reaction,
    increased F6P generation from glyceraldehyde-3-phosphate (G3P) leads to a faster depletion of triose phosphates on
    which the native carbon regeneration pathway is heavily reliant. In the case of the transaldolase reaction, a
    reverse flux of carbon regeneration is formed where sedulose-7-phosphate (S7P) and G3P are converted back to F6P and
    erythrose-4-phosphate (E4P) (Figure 6). Furthermore, the subsequent E4P and xylulose-5-phosphate (X5P), two key
    metabolites feeding into RuBP regeneration, are instead siphoned from carbon regeneration into the pentose phosphate
    pathway. Both overexpression strains lower the efficiency of carbon regeneration reactions in the native CBB cycle,
    likely causing the less robust growth observed here.
  </p>

  <span class="image half"
    ><img
      src="assets/images/proof-of-concept/figure-5.png"
      alt="Growth curves for wild type and single gene overexpression cells"
    /><span class="credit"> Figure 5: Growth curves for wild type and single gene overexpression cells. </span></span
  >

  <span class="image half"
    ><img src="assets/images/proof-of-concept/figure-6.png" alt="Native Calvin-Benson Cycle" /><span class="credit">
      Figure 6: Native Calvin-Benson Cycle.
    </span></span
  >

  <p>
    Based on these results, we would expect the double gene overexpression strain (<em>tal-fbp</em>) to show the same
    phenotype or an additive effect with worse growth. However, instead we observe improved growth compared to both
    single gene overexpression strains (Figure 7). This finding can be explained by the successful activation of our
    transaldolase pathway. With an even greater pool of F6P, transaldolase is thermodynamically favored to operate in
    the reverse as our pathway requires. While the native carbon regeneration pathway is more dependent on triose
    phosphates (C3), the redesigned transaldolase cycle depends more heavily on the hexose phosphate (C6) pools, which
    was also confirmed by computational modeling results. The robustness of the carbon regeneration by either the native
    pathway or the transaldolase pathway would thus depend on the abundance of either C3 or C6 carbon pools. Activation
    of our pathway might not have been observed in the transaldolase strain (<em>tal</em>) since there are not enough C6
    carbon pools available. With greater FBPase activity, however, a larger pool of hexose phosphates particularly in
    the form of F6P, favors the reverse transaldolase reaction, i.e. converting F6P and E4P to S7P and G3P. This
    establishes the transaldolase pathway for carbon regeneration. .
  </p>
  <p>
    This direct competition between SBPase and transaldolase also could explain why cells with our activated pathway do
    not show growth identical to WT (Figure 8). The transaldolase pathway is actively competing for intermediates with
    the native SBPase-driven carbon regeneration reactions, leading to a slightly less robust growth in comparison to
    the WT. Therefore, after deleting the SBPase gene in our overexpression strains (<em>tal-fbp</em>), we expect to see
    growth that is comparable or even outperforms WT depending on the abundance of phosphate sugars.
  </p>

  <span class="image half"
    ><img
      src="assets/images/proof-of-concept/figure-7.png"
      alt="Growth curves for tal, fbp, and tal-fbp overexpression cells"
    /><span class="credit"> Figure 7: Growth curves for tal, fbp, and tal-fbp overexpression cells. </span></span
  >

  <span class="image half"
    ><img
      src="assets/images/proof-of-concept/figure-8.png"
      alt="Growth curves for wild type and tal-fbp overexpression cells"
    /><span class="credit"> Figure 8: Growth curves for wild type and tal-fbp overexpression cells. </span></span
  >
</section>

<hr />
<bibliography />

<hr />
<explore pages="Team,Description" />
