<!-- Content -->
<section>
  <header class="main">
    <h1>Safety</h1>
  </header>

  <!-- Content -->
  <p class="fadeMe">
    Our project centers around creating genetically modified cyanobacteria strains. Although our selected strain is
    non-pathogenic and poses no direct threat to humans or animals, care must be taken any time an organism is
    genetically engineered. Furthermore, wet lab work inherently demands safety and security protocols; organisms must
    be contained within the lab, workers must be kept safe, waste must be properly disposed of, all of which demand a
    level of responsibility and risk management. The pandemic posed additional challenges for the collaborative efforts
    of the team and lab work, which we were able to navigate using online avenues and following our university
    regulations for research.
  </p>

  <hr class="major" />

  <h2 class="fadeMe">Safety in Project Design</h2>
  <p class="fadeMe">
    Safety and security begin with project design. Thus, all organisms used in our project fall under Biosafety Level 1
    (BSL-1).
  </p>
  <p class="fadeMe">
    <span class="image left"><img src="assets/images/pic03.jpg" alt="" /></span> We selected a non-pathogenic chassis in
    <em>Synechococcus elongatus </em>PCC 7942<em>. S. elongatus</em> PCC 7942 has no current evidence of cytotoxin
    production or cytotoxicity, and is considered a safe host for future biotechnological production
    <a href="#reference-1">(1)</a>. This freshwater cyanobacteria is not harmful to humans and is categorized as BSL-1,
    making it an extremely safe model organism for laboratory work, similar to the decision made by the
    <a href="http://2017.igem.org/Team:UCSC/Safety">2017 UCSC</a> team, who used the same model organism.
  </p>
  <p class="fadeMe">
    The cyanobacteria strain we selected is naturally competent and reliably transformable by exogenously added DNA
    <a href="#reference-2">(2)</a>. This strain is not, however, ideal for cloning of plasmids. We selected
    <em>Escherichia coli strain </em>DH10<em> </em>to clone and replicate our plasmids. This nonpathogenic strain is a
    derivative of <em>E. coli </em>K12<em>,</em> a lab strain of <em>E. coli</em> in Risk Group 1 commonly used in iGEM
    projects.
  </p>
  <p class="fadeMe">
    Our plasmids use antibiotic resistance markers to identify successful transformants, specifically for kanamycin,
    streptomycin and spectinomycin. Even though the organisms these markers were implemented into are not inherently
    pathogenic, antibiotic resistance genes are able to transfer between bacterial species, contributing to the
    ever-growing issue of widespread antibiotic resistance. In the wild, cyanobacterial blooms contribute to antibiotic
    gene transfer via conjugation <a href="#reference-3">(3)</a>. This factor, in combination with the general
    importance of containing genetically modified organisms within the lab, made it imperative to sterilize all waste
    before disposal and practice effective containment of organisms within the lab.
  </p>

  <hr class="major" />

  <h2 class="fadeMe">Safety in Lab Work</h2>
  <p class="fadeMe">
    As with any form of laboratory work, it is important to identify where hazards to health and safety may arise. From
    there, a scientist must take the precautions to both avoid accidents and know how to respond if they do occur.
  </p>
  <p class="fadeMe">
    <span class="image right"><img src="assets/images/pic03.jpg" alt="" /></span>
    All wet lab work was conducted in Xin Wang's lab within the Department of Microbiology at Miami University. All work
    was conducted under the guidance and supervision of either Dr. Wang or his graduate students Shaun Jiang and
    Shrameeta Shinde, who all regularly work with the techniques we used for this project in their regular studies on
    metabolic engineering with cyanobacteria. We are extremely grateful to this team for assisting with, teaching us,
    and explaining safety measures and proper protocols.
  </p>
  <p class="fadeMe">
    As a Biosafety Level 1 lab, there were clear guidelines that each student followed. Appropriate clothing (long
    pants, tied back hair, closed toe shoes) and safety equipment such as gloves were always worn when conducting
    experiments. No food or drink was allowed into the lab.
  </p>
  <p class="fadeMe">
    Most of the components used in wet lab work were safe. A major exception was the ethidium bromide used for DNA
    visualization during gel electrophoresis; this intercalating agent is hazardous due to its mutagenic properties. As
    such, ethidium bromide and its associated materials were always handled with gloves in a designated gel
    electrophoresis area. All equipment used in this area was considered contaminated with ethidium bromide, and thus
    items such as pipettes and ingredient containers are kept in this area and not handled unless gloves are worn.
  </p>
  <p class="fadeMe">
    Any other reagents used were for standard protocols and were commercially available from reputable suppliers. Many
    of our protocols and supply kits for plasmid work came from <a href="https://www.neb.com">NEB</a>, which has
    publicly available protocols detailing safe and efficient use of materials. Gloves were changed regularly to limit
    contamination and spread of any chemical or biological agents.
  </p>
  <p class="fadeMe">
    Our lab was also equipped with a variety of equipment for proper handling of microbial organisms. A vertical laminar
    airflow hood was used for all processes involving transformation or transfer of microbial organisms, limiting risks
    of contamination to the colonies and ourselves. All biological waste is sterilized via autoclave before being sent
    to designated university waste disposal services for the Microbiology Department. Similarly, all pipette tips,
    centrifuge tubes or other disposable waste is autoclaved and disposed of by university services. We further noted
    the location of important safety features such as the glass waste disposal, a fire extinguisher, a safety shower,
    and an eyewash station.
  </p>

  <hr class="major" />

  <h2 class="fadeMe">Safety During Covid-19</h2>
  <p class="fadeMe">
    <span class="image right"><img src="assets/images/pic03.jpg" alt="" /></span>
    With the emergence of the COVID-19 pandemic and the arising new strains, it was imperative to limit in-person
    contact and take precautions when it couldn't be avoided.
  </p>
  <p class="fadeMe">
    Many of our team meetings occurred over Google Meets, allowing team members to communicate and organize while
    preventing spread of disease between each other. When in person meetings did occur, due to local allowances
    permitting it, team members sat at least six feet apart and often wore masks. All wet lab workers, which totalled 3
    student team members, maintained social distancing during lab work and had full vaccination status.
  </p>
  <p class="fadeMe">
    In consideration of the alarming spread of new variants, Miami University reimplemented the requirement to wear a
    mask indoors on campus just as wet lab work was beginning. Faculty and students were (and still are) expected to
    follow this rule regardless of vaccination status. When participating in wet lab activities, all 3 student workers
    wore a mask and were expected to self-monitored for potential symptoms and stay home if there was cause for concern.
  </p>

  <hr class="major" />

  <!-- References -->
  <h3 class="fadeMe">References</h3>
  <p id="reference-1" class="fadeMe">
    1. Klemencic M, Nielsen AZ, Sakuragi Y, Frigaard N-U, Celesnik H, Jensen PE, Dolinar M. 2017. 13 - Synthetic biology
    of cyanobacteria for production of biofuels and high-value products, p. 305â€“325. In Gonzalez-Fernandez, C, Munoz, R
    (eds.), Microalgae-Based Biofuels and Bioproducts. Woodhead Publishing.
    <a href="https://doi.org/10.1016/B978-0-08-101023-5.00013-3">https://doi.org/10.1016/B978-0-08-101023-5.00013-3</a>.
  </p>
  <p id="reference-2" class="fadeMe">
    2. Golden S. Synechococcus elongatus PCC 7942. Accessed at:
    <a href="https://genome.jgi.doe.gov/portal/synel/synel.home.html"
      >https://genome.jgi.doe.gov/portal/synel/synel.home.html</a
    >.
  </p>
  <p id="reference-3" class="fadeMe">
    3. Wang Z, Chen Q, Zhang J, Guan T, Chen Y, Shi W. 2020. Critical roles of cyanobacteria as reservoir and source for
    antibiotic resistance genes. Environment International 144:106034.
    <a href="https://doi.org/10.1016/j.envint.2020.106034">https://doi.org/10.1016/j.envint.2020.106034</a>.
  </p>
</section>
